smash/core/signal_analysis/signatures/_standardize.py

```diff
@@ -10,7 +16,9 @@
 import warnings
 
 
-def _standardize_signatures_sign(sign: str | list[str] | None) -> tuple[list[str]]:
+def _standardize_compute_signatures_sign(
+    sign: str | list[str] | None,
+) -> tuple[list[str]]:
```

Comment:
```suggestion
    sign: str | ListLike | None,
) -> tuple[ListLike]:
```
A moins qu'on puisse pas utiliser ce tuple ?